examples/removeVertexgiventestcase.cpp

```diff
@@ -0,0 +1,91 @@
+#include <iostream>
+#include <string>
+#include "../include/GraphMatrix.h"
+
+using namespace Appledore;
+
+// TestVertex class implementation
+class TestVertex
```

Comment:
If you are creating a custom vertex for ``GraphMatrix`` you should inherit your custom vertex type from the ``GraphVertex`` class present
```suggestion
class TestVertex: public GraphVertex
```